Source/Diagnostics/BTDiagnostics.cpp

```diff
@@ -139,17 +163,81 @@ BTDiagnostics::InitializeFieldBufferData ( int i_buffer , int lev)
 {
     auto & warpx = WarpX::GetInstance();
     // 1. Lab-frame time for the i^th snapshot
+    m_t_lab[i_buffer] = i_buffer * m_dt_snapshots_lab;
+    // 2. Define domain in boosted frame at level, lev
+    amrex::RealBox diag_dom;
```

Comment:
Maybe worth double-checking for the next min/max loop in moving window scenarios:
the default values for the `RealBox` are `0` and `-1`: https://github.com/AMReX-Codes/amrex/blob/9b2172017f8aa6aea00bda090e7c93e7919ff6eb/Src/Base/AMReX_RealBox.H#L23
Maybe setting it to [`std::numeric_limits`](https://en.cppreference.com/w/cpp/types/numeric_limits/lowest) `max`/`lowest` here avoids any potential corner-cases.

Reply:
Offline discussed: Ah no, this is not a `x = min(x,y);` pattern - sorry, overlooked :)

Reply:
Here we set the lo and hi and not find the min/max by comparing with previous idim. As discussed offline, I will add comments here to better explain this.